### PR TITLE
updated to wpi 2023 and started april tags a little bit

### PR DESCRIPTION
**Pull request by pixelmonaskarion**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java
Comment (Irishumanoid) [inline src/main/java/frc/robot/Robot.java:37]: Can someone confirm this has been tested and is functional before it is merged into main?

File: src/test/java/WaypointParserTest.java
Comment (Irishumanoid) [inline src/test/java/WaypointParserTest.java:20]: why are you instantiating this?

File: src/main/java/frc/robot/Robot.java
Comment (pixelmonaskarion) [reply src/main/java/frc/robot/Robot.java:37]: It has been tested, it doesn't break anything but it doesn't work quite yet either

Comment (Irishumanoid) [reply src/main/java/frc/robot/Robot.java:37]: then let's wait until tomorrow to debug and then merge

File: src/test/java/WaypointParserTest.java
Comment (pixelmonaskarion) [reply src/test/java/WaypointParserTest.java:20]: I changed your test code a bit, fixed file loading. the problem right now is that you cannot compare with that datatype, I was starting to fix that with that line

**Review by Irishumanoid — APPROVED** [review]

File: src/test/java/WaypointParserTest.java
Comment (garrettwitowski) [inline src/test/java/WaypointParserTest.java:26]: As a rule of thumb we should not be submitting/merging commented out code, as this reduces readability and can be confusing to others. Also, any unit tests should be fully functional prior to submitting/merging.

**Pull request merged**
